.github/scripts/withOnyxMigration.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+RED='\033[0;31m'
+ORANGE='\033[0;33m'
+GREEN='\033[0;32m'
+BOLD_WHITE='\033[1;37m'
+
+# Set IFS to newline to properly handle file names with spaces.
+# This makes 'for' loop treat each line as a separate item.
+IFS=$'\n'
+
+# Fetch the main branch
+git fetch origin main --no-tags --depth=1
+
+# Get the list of modified and added files
+modified_files=$(git diff --name-only --diff-filter=AM origin/main HEAD -- '*.ts' '*.tsx')
+
+# Count instances of 'withOnyx' on main branch and current branch
+count_main=0
+count_current=0
+
+echo -e "\nWe're gradually moving from 'withOnyx' to 'useOnyx'. 'useOnyx' hook simplifies the code, offers performance benefits and improves TypeScript compilation time."
+echo -e "For more details, see https://expensify.slack.com/archives/C01GTK53T8Q/p1725905735105989.\n"
+echo -e "${BOLD_WHITE}If you are blocked by this on something that is very urgent, you can always ignore this check and merge with it failing.\n"
```

Comment:
```suggestion
echo -e "${BOLD_WHITE}If this check is blocking you on something that is very urgent, you can always ignore it.\n"
```

maybe just a simpler and shorter sentence is better?